nvtabular/ops/categorify.py

```diff
@@ -740,7 +747,7 @@ def _encode(name, storage_name, path, gdf, cat_cache, na_sentinel=-1, freq_thres
         value.index.name = "labels"
         value.reset_index(drop=False, inplace=True)
 
```

Comment:
May need to raise an error if `freq_threshold>0` **and**  `search_sorted` is `True`.  However, it is probably best to catch this problem quickly in the `Categorify` initializer.